modules/tinymce/src/plugins/textpattern/test/ts/browser/FindInlinePatternTest.ts

```diff
@@ -13,14 +13,22 @@ import Theme from 'tinymce/themes/silver/Theme';
 UnitTest.asynctest('textpattern.browser.FindInlinePatternTest', (success, failure) => {
   Theme();
 
-  const inlinePatterns = Settings.getPatternSet({ textpattern_patterns: [
-    { start: '*', end: '*', format: 'italic' },
-    { start: '**', end: '**', format: 'bold' },
-    { start: '***', end: '***', format: [ 'bold', 'italic' ] }, // due to priority this will never be used
-    { start: '', end: 'brb', cmd: 'mceInsertContent', value: 'be right back' },
-    { start: 'irl', end: '', cmd: 'mceInsertContent', value: 'in real life' },
-    { start: 'asap', replacement: 'as soon as possible' }
-  ] }).inlinePatterns;
+  const mockEditor = {
+    getParam: () => (
```

Comment:
This looks incorrect, as `getParams` would never return this... it instead would return the array specified by `textpattern_patterns`. That likely means there's a bug elsewhere as well.

Reply:
Did I do the conversion to getParam wrong, or do you mean that there's an error in the original code that I happily transferred over?

Reply:
There were 2 issues:

1. This test file was updated incorrectly and I'm not sure how tests are passing because of it.
2. There was a bug in the `getParam` call which I commented on elsewhere (it was using `object` instead of `array` as the type).

So to be clear, the text pattern plugin expects an array to be returned here, but it's currently returning an object.